app/views/users/dashboard.html.erb

```diff
@@ -0,0 +1 @@
+<a></a>
```

Comment:
I may be missing something but is this all there is on the `/dashboard`? The user story requires user profile information. And I haven't seen a change in the `layouts/application.html.erb` that has anything that will change a logout button instead of login when a user is logged in.